aws/opsworks_layers.go

```diff
@@ -292,12 +297,15 @@ func (lt *opsworksLayerType) Read(d *schema.ResourceData, client *opsworks.OpsWo
 		d.Set("short_name", layer.Shortname)
 	}
 
-	if v := layer.CustomJson; v == nil {
-		if err := d.Set("custom_json", ""); err != nil {
-			return err
+	v := layer.CustomJson
+	if v == nil {
+		d.Set("custom_json", "")
+	} else {
+		policy, err := structure.NormalizeJsonString(*v)
+		if err != nil {
+			return errwrap.Wrapf("policy contains an invalid JSON: {{err}}", err)
```

Comment:
Nitpick: As this error is going to be raised straight to the user I don't see the point in using errwrap here.

Reply:
Updated, thanks for the explanation btw!